### ISSUE TITLE
Multiflagellar spermatozoa

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7616]
**Preferred term label:**
Multiflagellar spermatozoa

**Synonyms**
Multi-tailed spermatozoa 

**Definition (free text, please give PubMed ID)**
Spermatozoa with multiple flagella attached to the sperm head.

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
243060 
 PMID: 21733974   2/2

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
added
